src/concurrency/Color.java

```diff
@@ -0,0 +1,86 @@
+package concurrency;
+
+public enum Color {
```

Comment:
differentiating foreground vs background would be good, so you don't have to remember later...